jinja/index.html

```diff
@@ -16,6 +16,32 @@
             <h3 class="col-12 mt-5 mb-5 grey-text" style="font-family: 'Roboto', sans-serif !important;">A cross-platform Python library for differentiable programming of quantum computers. Train a quantum computer the same way as a neural network.</h3>
         </div>
 
+        <div class="row pb-5 align-items-center justify-content-center">
+            <div class="m-3 card-image">
+                <a target="_blank"
+                   href="https://hacktoberfest.digitalocean.com/">
+                    <img width="200px" src="img/Hacktoberfest_Wallpaper_desktop-02.png" />
+                </a>
+            </div>
+            <div class="m-3 text-center align-items-center">
+                <h3 class="card-title"></i>
+                    <strong>PennyLane is participating in Hacktoberfest!</a></strong>
+                </h3>
+                <p class="black-text">
+                    <br>Join us on GitHub on October 1st-31st. Contibute with at least 4 approved
+                    <br>pull requests in participating projects and register on the   
+                    <br>Hacktoberfest website to get the chance to win a t-shirt or plant a tree!
```

Comment:
> to get the chance to win a t-shirt

Do ppl. not get a T-shirt if they have at least 4 approved PRs?

Reply:
They have about 70k t-shirts. Around 50k will be first come first serve. The rest will be merit-based. So there is a chance that someone may do the 4PRs but get no t-shirt.